services/QuillLMS/spec/controllers/teacher_infos_controller_spec.rb

```diff
@@ -14,7 +14,6 @@
 require 'rails_helper'
 
 describe TeacherInfosController do
-  let(:user) { create(:user) }
```

Comment:
Something must have come up in the tests that's not obvious from just reading the code, but I don't see a reason why it should be necessary to move the `let(:user)` from this section into each separate `describe`.  Not a big deal either way.

Reply:
I think I was anticipating an issue with how we use `current_user` in this controller and how it would affect the tests with these changes, but that's not actually the case-- I'll revert this